packages/react/list.tsx

```diff
@@ -112,7 +112,7 @@ const List = ({
         };
 
         if (attribute.relation) {
-          column["identifier"] = resolveByDot("relation.resolveTo", attribute);
+          column["identifier"] = attribute.identifier;
```

Comment:
@rajeshn95 I am not sure, where we are using only resolveTo

Can we make this:

```
          column["identifier"] = resolveByDot("relation.resolveTo", attribute) || attribute.identifier;
```

Or differently, why can't we continue with resolveTo? Why are we changing to identifier?

Reply:
@rjvim Needed this identifier in filterBy while making API call, will change it.